examples/plot_text_classification.py

```diff
@@ -0,0 +1,169 @@
+"""
+scikit-learn model cards
+--------------------------------------
+
+This guide demonstrates how you can make the best of this package to host text
+classification models.
+"""
+
+# %% Imports
+# =======
+# First we will import everything required for the rest of this document.
+
+import pickle
+from pathlib import Path
+from tempfile import mkdtemp, mkstemp
+
+import pandas as pd
+import sklearn
+from sklearn.datasets import fetch_20newsgroups
+from sklearn.feature_extraction.text import CountVectorizer, TfidfTransformer
+from sklearn.metrics import (
+    ConfusionMatrixDisplay,
+    accuracy_score,
+    classification_report,
+    confusion_matrix,
+    f1_score,
+)
+from sklearn.model_selection import train_test_split
+from sklearn.naive_bayes import MultinomialNB
+from sklearn.pipeline import Pipeline
+
+from skops import card, hub_utils
+
+# %% Data
+# ====
+# We will use 20 newsgroups dataset from sklearn. We will define categories of
+# this dataset ourselves as targets are integers.
```

Comment:
scikit-learn returns `target_names` which we can use, instead of hard copying them here.